### PR TITLE
Add Reaper64 Notarized recipes

### PR DESCRIPTION
**Pull request by apizz**
Resubmit of #11.  Along the lines of #9 & #10, this PR adds dedicated recipes for the notarized Reaper64 app version, since the download URL is different

### CODE REVIEW

**Comment by apizz** [comment]
👋

File: Reaper/Reaper64Notarized.munki.recipe [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+			<key>name</key>
+			<string>%NAME%</string>
+			<key>unattended_install</key>
+			<true/>
+		</dict>
+	</dict>
+	<key>MinimumVersion</key>
+	<string>0.6.1</string>
+	<key>ParentRecipe</key>
+	<string>com.github.orbsmiv.download.Reaper64</string>
```
[the remark this answers is not in the document]
Comment (apizz) [reply Reaper/Reaper64Notarized.munki.recipe]: Yes, my mistake

**Comment by apizz** [comment]
Much appreciated, thanks!

File: Reaper/Reaper64Notarized.download.recipe [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	<key>Identifier</key>
+	<string>com.github.orbsmiv.download.Reaper64Notarized</string>
+	<key>Input</key>
+	<dict>
+		<key>DOWNLOAD_URL</key>
+		<string>https://www.reaper.fm/download.php</string>
+		<key>NAME</key>
+		<string>Reaper64</string>
+		<key>RE_PATTERN</key>
+		<string>(?P&lt;url&gt;files\/6\.x\/reaper(?P&lt;version&gt;\d+\.*)_x86_64_catalina\.dmg)</string>
```
[the remark this answers is not in the document]
Comment (orbsmiv) [reply Reaper/Reaper64Notarized.download.recipe]: Sorry – I've tried to commit the suggested changes to your repo but clearly this didn't work :-)
Please can you hit the _Commit suggestion_ button if you're happy?

[the remark this answers is not in the document]
Comment (apizz) [reply Reaper/Reaper64Notarized.download.recipe]: Sorry, missed that.  Thanks for the prod

**Comment by apizz** [comment]
Done!  Thanks for the help.

File: Reaper/Reaper64Notarized.munki.recipe (hunk 1 above)
[the remark this answers is not in the document]
Comment (orbsmiv) [reply Reaper/Reaper64Notarized.munki.recipe]: @apizz please can you hit the _Commit suggestion_ button (or batch the changes) if you're happy?

[the remark this answers is not in the document]
Comment (apizz) [reply Reaper/Reaper64Notarized.munki.recipe]: I thought I did ...

[the remark this answers is not in the document]
Comment (apizz) [reply Reaper/Reaper64Notarized.munki.recipe]: I committed both suggestions, should be listed

Screen Shot 2020-01-21 at 10 29 16 AM

**Pull request merged**